spec/system/task_spec.rb

```diff
@@ -1,24 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
+  let(:task) { FactoryBot.create(:task, project_id: project.id) }
```

Comment:
Factoryに`association`を設定すると省略可能です。
```suggestion
  let(:task) { create(:task) }
```

Reply:
d7ade86012a69a6ca005e127447b4f925b097a1b で表記を省略しました。